packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -29,7 +29,7 @@ export class RouterliciousRestWrapper extends RestWrapper {
         private readonly getAuthorizationHeader: AuthorizationHeaderGetter,
         private readonly useRestLess: boolean,
         baseurl?: string,
-        defaultQueryString: Record<string, unknown> = {},
+        defaultQueryString: querystring.ParsedUrlQueryInput = {},
```

Comment:
The client build started to fail after some protocol changes got pushed to the main. Specifically #7522